sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -118,10 +119,10 @@ def upload_blob_to_url(
     :rtype: dict(str, Any)
     """
     with BlobClient.from_blob_url(blob_url, credential=credential) as client:
-        return client.upload_blob(data=data, blob_type=BlobType.BlockBlob, **kwargs)
+        return client.upload_blob(data=data, blob_type=BlobType.BLOCKBLOB, **kwargs)  # type: ignore [no-any-return]
```

Comment:
azure\storage\blob\__init__.py:122: error: Returning Any from function declared to return "Dict[str, Any]"  [no-any-return]

Reply:
Does this mean `BlobClient.upload_blob` is typed wrong? Or `from_blob_url`?

Reply:
Wow would really think it would be smart enough. Can you try adding a `client: BlobClient` on line 121 instead of the cast. If that doesn't work, this is fine.

Reply:
Yeah, couldn't get that work, even tried parenthesizing:
`client: BlobClient`
`(client: BlobClient):`